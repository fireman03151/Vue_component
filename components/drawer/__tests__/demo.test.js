import demoTest from '../../../tests/shared/demoTest'

demoTest('drawer', { skip: ['from-drawer.md'] })
