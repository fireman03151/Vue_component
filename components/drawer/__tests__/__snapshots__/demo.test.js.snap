// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/drawer/demo/basic-right.md correctly 1`] = `
<div><button type="button" class="ant-btn ant-btn-primary"><span>Open</span></button>
  <!---->
</div>
`;

exports[`renders ./components/drawer/demo/form-in-drawer.md correctly 1`] = `
<div><button type="button" class="ant-btn ant-btn-primary"><i aria-label="icon: plus" class="anticon anticon-plus"><svg viewBox="64 64 896 896" data-icon="plus" width="1em" height="1em" fill="currentColor" aria-hidden="true" focusable="false" class="">
        <path d="M482 152h60q8 0 8 8v704q0 8-8 8h-60q-8 0-8-8V160q0-8 8-8z"></path>
        <path d="M176 474h672q8 0 8 8v60q0 8-8 8H176q-8 0-8-8v-60q0-8 8-8z"></path>
      </svg></i><span>New account</span></button>
  <!---->
</div>
`;

exports[`renders ./components/drawer/demo/multi-level-drawer.md correctly 1`] = `
<div><button type="button" class="ant-btn ant-btn-primary"><span>Open</span></button>
  <!---->
</div>
`;

exports[`renders ./components/drawer/demo/placement.md correctly 1`] = `
<div>
  <div class="ant-radio-group ant-radio-group-outline ant-radio-group-default" style="margin-right: 8px;"><label class="ant-radio-wrapper"><span class="ant-radio"><input type="radio" class="ant-radio-input" value="top"><span class="ant-radio-inner"></span></span><span>top</span></label><label class="ant-radio-wrapper"><span class="ant-radio"><input type="radio" class="ant-radio-input" value="right"><span class="ant-radio-inner"></span></span><span>right</span></label><label class="ant-radio-wrapper"><span class="ant-radio"><input type="radio" class="ant-radio-input" value="bottom"><span class="ant-radio-inner"></span></span><span>bottom</span></label><label class="ant-radio-wrapper ant-radio-wrapper-checked"><span class="ant-radio ant-radio-checked"><input type="radio" class="ant-radio-input" value="left"><span class="ant-radio-inner"></span></span><span>left</span></label></div> <button type="button" class="ant-btn ant-btn-primary"><span>Open</span></button>
  <!---->
</div>
`;

exports[`renders ./components/drawer/demo/user-profile.md correctly 1`] = `
<div>
  <div class="ant-list ant-list-split ant-list-bordered">
    <div class="ant-spin-nested-loading">
      <div class="ant-spin-container">
        <div class="ant-list-item">
          <div class="ant-list-item-meta">
            <div class="ant-list-item-meta-avatar"><span class="ant-avatar ant-avatar-circle ant-avatar-image"><img src="https://gw.alipayobjects.com/zos/rmsportal/BiazfanxmamNRoxxVxka.png"></span></div>
            <div class="ant-list-item-meta-content">
              <h4 class="ant-list-item-meta-title"><a href="https://www.antdv.com/">Lily</a></h4>
              <div class="ant-list-item-meta-description">Progresser AFX</div>
            </div>
          </div>
          <ul class="ant-list-item-action">
            <li><a>View Profile</a></li>
          </ul>
        </div>
        <div class="ant-list-item">
          <div class="ant-list-item-meta">
            <div class="ant-list-item-meta-avatar"><span class="ant-avatar ant-avatar-circle ant-avatar-image"><img src="https://gw.alipayobjects.com/zos/rmsportal/BiazfanxmamNRoxxVxka.png"></span></div>
            <div class="ant-list-item-meta-content">
              <h4 class="ant-list-item-meta-title"><a href="https://www.antdv.com/">Lily</a></h4>
              <div class="ant-list-item-meta-description">Progresser AFX</div>
            </div>
          </div>
          <ul class="ant-list-item-action">
            <li><a>View Profile</a></li>
          </ul>
        </div>
      </div>
    </div>
  </div>
  <!---->
</div>
`;
